### PR TITLE
Max - Test one

### PR DESCRIPTION
**Pull request by FireMugen**
Test one complete.

### CODE REVIEW

File: test-one/src/index-test.js [hunk 1]
```diff
@@ -1,13 +1,36 @@
...
+    "Boss 52 53 Boss Hog 56 Boss 58 59 BossHog 61 62 Boss 64 Hog Boss 67 68 Boss Hog 71 Boss 73 74 BossHog " + 
+    "76 77 Boss 79 Hog Boss 82 83 Boss Hog 86 Boss 88 89 BossHog 91 92 Boss 94 Hog Boss 97 98 Boss Hog";
   logger.log = jest.fn();
-
   await go();
-
   expect(logger.log).toBeCalledWith(expectedOutput);
 });
+
+test(`should return the number when it is not a multiple of 3 or `, () => {
```
Comment (danwfeeney) [inline test-one/src/index-test.js:17]: 👍 for effective tests

File: test-one/src/index.js [hunk 2]
```diff
@@ -1,14 +1,23 @@
...
+  return num % mod === 0;
+}
 
-  const welcomeMessage =
-    "Congrats, you're up and running! Welcome to the first moshtix code challenge! " +
-    "Please replace this message with your solution and feel free to add " +
-    "or modify other files as necessary. Good luck : )";
+export const main = (num) => {
+  switch(true) {
+    case isDivisibleBy(num, 15): return "BossHog";
```
Comment (danwfeeney) [inline test-one/src/index.js:9]: 👍

**Review by danwfeeney — COMMENTED** [review]
Nice job, this is all pretty clean. What inspired you to change tack after your initial submission?

**Comment by FireMugen** [comment]
Thanks!  I wanted to improve on my solution,
I saw some closed pull requests had comments and thought i'd try to make my code more robust.
Also I wanted to try and refactor it without loops if possible. 
Splitting up my functions helped a lot with testing and I probably should have approached it that way from the beginning.

**Pull request closed without merging**